### PR TITLE
[TASK] HubSpot features and bugfixes

### PR DESCRIPTION
**Pull request by pixelmatseriks**

### CODE REVIEW

File: Classes/Domain/Repository/Hubspot/AccountInformationRepository.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * Repository for Hubspot custom object definitions.
+ */
+class AccountInformationRepository extends AbstractHubspotRepository
+{
+    /**
+     * @param Factory|null $factory
+     */
+    public function __construct(Factory $factory = null)
+    {
+        parent::__construct($factory);
```
Comment (andreaskienast) [inline Classes/Domain/Repository/Hubspot/AccountInformationRepository.php]: The overridden constructor looks superfluous to me.

File: Classes/Domain/Repository/Hubspot/AccountInformationRepository.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    /**
+     * Get account details for a HubSpot account.
+     *
+     * @return array
```
Comment (andreaskienast) [inline Classes/Domain/Repository/Hubspot/AccountInformationRepository.php]: We don't need `@return` annotations if the returned type is declared already (`: array`).

File: Classes/Hubspot/Resources/AccountInformation.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+class AccountInformation extends Resource
+{
+    protected const ENDPOINT_PREFIX = 'https://api.hubapi.com/account-info/v3/';
+
+    /**
+     * AccountInformation constructor.
+     */
+    public function __construct(Client $client)
+    {
+        parent::__construct($client);
```
Comment (andreaskienast) [inline Classes/Hubspot/Resources/AccountInformation.php]: The overridden constructor looks superfluous to me.

**Review by andreaskienast — CHANGES_REQUESTED** [review]

**Review by mabolek — APPROVED** [review]
Looks like some good improvements and fixes. ❤️

**Pull request merged**